### PR TITLE
Added docker file for project

### PR DESCRIPTION
**Pull request by zhikiri**
Added docker file with tiny explanation in readme file.
Please, check if provided configuration is okay.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM node:8.6
+
+# Create app directory
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+# Install app dependencies
+COPY package.json /usr/src/app/
+COPY yarn.lock /usr/src/app/
```
Comment (butlerx) [inline Dockerfile]: could you add a cmd that runs the script?

Comment (zhikiri) [reply Dockerfile]: Sure, but I thought better to call with `docker run`. And if you will use docker-compose, there is `command` property in yml file. But if you want to put here, no problem :)

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+ADD . /usr/src/app
+
+RUN yarn install
```
Comment (butlerx) [inline Dockerfile]: can we move this abouve add

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+CMD ["node", "index.js", "ldapsearch -D cn=root,ou=ldap,o=redbrick -xLLL -y /etc/ldap.secret objectClass uid gidNumber > entry.ldif"]
```
Comment (butlerx) [inline Dockerfile]: this shoudl just be `CMD ["node", "index.js"]`

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,15 @@
+FROM node:8.6
+
+# Create app directory
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+# Install app dependencies
+COPY package.json /usr/src/app/
```
Comment (butlerx) [inline Dockerfile]: could you copy the package and yarn at the same tim

File: README.md [hunk 5]
```diff
@@ -9,3 +9,13 @@ yarn
 ldapsearch -D cn=root,ou=ldap,o=redbrick -xLLL -y /etc/ldap.secret objectClass uid gidNumber > entry.ldif
 node .
 ```
+
+## docker
+
+First of all, you need to build a container:
+
+`docker build -t user-vhost-generator`
```
Comment (butlerx) [inline README.md]: this shoudl have a `.` at the end

File: README.md [hunk 6]
```diff
@@ -9,3 +9,13 @@ yarn
...
+
+Next, install project dependencies:
+
+`docker run -v ${PWD}:/usr/src/app -t user-vhost-generator yarn install`
```
Comment (butlerx) [inline README.md]: is the yarn install still needed

File: Dockerfile (hunk 2 above)
Comment (zhikiri) [reply Dockerfile]: I've changed it. Check that out

**Pull request merged**